components/steps/w-step.vue

```diff
@@ -1,55 +1,78 @@
 <template>
-  <div :class="{
-      [c.step]: true,
-      [c.stepVertical]: vertical,
-      [c.stepVerticalLeft]: vertical && left,
-      [c.stepVerticalRight]: vertical && !left,
-      [c.stepHorizontal]: !vertical
-  }">
-    <div v-if="!vertical" :class="{
-      [c.stepLine]: true,
-      [c.stepLineHorizontal]: !vertical,
-      [c.stepLineHorizontalLeft]: true,
-      [c.stepLineIncomplete]: !active && !complete,
-      [c.stepLineComplete]: active || complete
-    }" />
-    <div :aria-current="active ? 'step' : undefined" :class="{
-      [c.stepDot]: true,
-      [c.stepDotVertical]: vertical,
-      [c.stepDotVerticalLeft]: vertical && left,
-      [c.stepDotVerticalRight]: vertical && !left,
-      [c.stepDotHorizontal]: !vertical,
-      [c.stepDotIncomplete]: !(active || complete),
-      [c.stepDotActive]: active,
-      [c.stepDotComplete]: complete
-    }">
-      <svg role="img" :aria-label="complete ? '✓' : '⍻'" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
-        <path fill="currentColor" d="M13.007 4.447a.75.75 0 01.046 1.06l-5.5 6a.75.75 0 01-1.083.023l-3-3a.75.75 0 011.06-1.06l2.446 2.446 4.971-5.423a.75.75 0 011.06-.046z" />
+  <div
+    :class="{
+      [ccStep.step]: true,
```

Comment:
Would really want to rewrite this as computeds... 😅

Reply:
yeah ofc, I fix! There's a mix of how it's done in the repo😅 But then we'll do it like that everywhere